March On map climate strike style iterations

mikelmaron opened the pull request.

mikelmaron committed:
climate strike event styling

mikelmaron committed:
make icon 33% bigger

mikelmaron committed:
iconoffset for climate map

mikelmaron committed:
exclude actionnetwork events from closest search

jillh510 commented on map.js:
```diff
@@ -417,10 +437,11 @@ const app = new Vue({
       const withDistance = this.features.map(function(feature) {
         const coords = feature.geometry.coordinates;
-        const distance = _this.distance(loc.latitude, loc.longitude, coords[1], coords[0]);
+        const distance = (typeof feature.properties.source != "undefined" && feature.properties.source != "actionnetwork" ? _this.distance(loc.latitude, loc.longitude, coords[1], coords[0]) : 100000);
```
this will change all the maps; please only make the change if it's on the climate page.

mikelmaron replied:
changed

jillh510 commented on map.js:
```diff
@@ -390,7 +400,16 @@ const app = new Vue({
         return;
       }
 
+      var isClimatePage = false;
```
is there a better solution than having this code twice?

mikelmaron replied:
dedup'd

mikelmaron committed:
dedup isClimatePage; only filter distance if isClimatePage

mikelmaron commented:
@jillh510 addressed your comments, thanks. this code definitely needs refactoring -- will look into this on the other side of the climate strike event.
